rca/schools/migrations/0006_adds_key_details_fields.py

```diff
@@ -0,0 +1,106 @@
+# Generated by Django 2.2.12 on 2020-12-11 12:41
```

Comment:
Nitpick: May be too late if this has been merged elsewhere, but would it be possible to commit fewer migration?